.github/workflows/auto_assign_accebility_ticket.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto-Assign Accessibility Tickets
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  auto-assign:
+    if: github.event.label.name == 'accessibility'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - name: Assign ticket to axe specialist
+      run: |
+        gh issue edit ${{ github.event.issue.number }} --add-assignee "IsraelleHub"
```

Comment:
Minor nitpick but FYI if the command is just one line, you can just do `run: gh issue edit ${{ github.event.issue.number }} --add-assignee "IsraelleHub"` without the `|` and newline.